tcptraceroute: T2723: Some repairs have been made

jack9603301 opened the pull request:
- Support VRF
- Add dependency fixed tcptraceroute6
- Improve command line options, support execution without port

c-po commented:
Your commit message is well - very not like it should be. YOu combine two non relevant changes into one commit which is bad practice - see https://docs.vyos.io/en/latest/contributing/development.html#writing-good-commit-messages.

Please split to two commits.

c-po commented:
The TCP port in your implementation is mandatory - but in reality it is optional, please adjust your XML definition to make the port optional.

In addition your help string `Perform operations on TCP ports` should better be renamed to "TCP port to connect to for path tracing".

jack9603301 committed:
tcptraceroute: T2723: Fix the problem that tcptraceroute6 cannot be executed

jack9603301 committed:
tcptraceroute: T2723: Support VRF

jack9603301 committed:
tcptraceroute: T2723: Improve command line options, support execution without port

Because the ports of tcptraceroute and tcptraceroute6
do not need to be specified in this feature,
 modify the command line syntax

jack9603301 committed:
tcptraceroute: T2723: Modify port description

In response to #522 request, modify `Perform operations
on TCP ports` to `TCP port to connect to for path tracing`
